mypy/semanal.py

```diff
@@ -84,10 +84,20 @@
 T = TypeVar('T')
 
 
-# Inferred value of an expression.
-ALWAYS_TRUE = 0
-ALWAYS_FALSE = 1
-TRUTH_VALUE_UNKNOWN = 2
```

Comment:
Minor comments: uniform prefix will make it slightly easier to search and possibly replace, as will happen eventually when it will become an Enum. Not that TRUTH_VALUE_UNKNOWN is a hard name to search :)

How about giving this type a dedicated alias until then? making it under the namespace `class Truth: ...` already, before its values gets spread any further.